templates/terraform_aws_provider/provider.tf

```diff
@@ -1,3 +1,3 @@
-provider "{{ generateId() }}" {
+provider "aws" {
```

Comment:
the template `aws_provider` is updated here but it has been deleted from the file `index.json`